### PR TITLE
101681462 - Polish Left Nav according specs

### PR DESCRIPTION
**Pull request by ckrampanis-globant**

### CODE REVIEW

File: src/scripts/components/header.jsx [hunk 1]
```diff
@@ -110,21 +111,21 @@ module.exports = React.createClass({
                                 <li onClick={ 
                                     this.goTo.bind(null,'/')                                   
                                 }>
-                                    Expenses <span className="plus">+</span>
+                                    Expenses <span className="plus"><span className="more">+</span></span>
```
Comment (amilcar-infante) [inline src/scripts/components/header.jsx]: Is it possible to use a line for each span to help readability?

File: src/scripts/components/header.jsx [hunk 2]
```diff
@@ -17,6 +17,7 @@ module.exports = React.createClass({
     toggleMenu: function () {
         this.setState({ displayMenuNav: !this.state.displayMenuNav });
         $('.view-container, .react-menu-nav, .header').toggleClass('menu-open');
+        $('.content').toggleClass('not-active');
```
Comment (amilcar-infante) [inline src/scripts/components/header.jsx:20]: :+1:

File: src/styles/main.css [hunk 3]
```diff
@@ -63,12 +63,13 @@ li {
     -webkit-transition-property: height;
     transition-delay: 0.5s;
     -webkit-transition-delay: 0.5s;
-    -webkit-box-shadow: inset -5px 0px 13px 0px rgba(0,0,0,0.75);
+/*    -webkit-box-shadow: inset -5px 0px 13px 0px rgba(0,0,0,0.75);
```
Comment (amilcar-infante) [inline src/styles/main.css]: Remove commented rules

**Comment by amilcar-infante** [comment]
Minor comments, then LGTM

File: src/scripts/components/header.jsx (hunk 1 above)
Comment (ckrampanis-globant) [inline src/scripts/components/header.jsx]: fixed

File: src/styles/main.css (hunk 3 above)
Comment (ckrampanis-globant) [inline src/styles/main.css]: Removed

**Pull request merged**